src/components/Previewer.svelte

```diff
@@ -1,15 +1,34 @@
 <script>
-  import IssueDetails from "./IssueDetails.svelte";
-  import LanguageSelector from "./LanguageSelector.svelte";
-  import TemplateSelector from "./TemplateSelector.svelte";
-  import TemplatePreview from "./TemplatePreview.svelte";
-  import AdditionalFields from "./AdditionalFields.svelte";
-  import CopyTemplate from "./CopyTemplate.svelte";
+  import { languages } from "../templates";
+  import { buildBugUrl, buildSiteUrl } from "../helpers/utils";
+
+  export let selectedLanguage;
+  export let selectedTemplate;
+  export let issue;
+
+  const getTemplate = (languages, lng, tpl) => {
+    if (lng && tpl) {
+      return languages[lng].templates[tpl];
+    }
+  };
+
+  $: tpl = getTemplate(languages, selectedLanguage, selectedTemplate) || {};
```

Comment:
it's svelte syntax ? ` $:`

Reply:
yeah, this statement will rerun if one of the values changes https://svelte.dev/examples#reactive-declarations